widget/embedded/src/utils/routing.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
It maybe takes sometime but to be closer to what terms industry uses, we should consider to use `quote` instead of `route`. Please rename whatever you can to `quote`.